docs/admin/open-in-coder.md

```diff
@@ -0,0 +1,29 @@
+# Open in Coder Button
+
+> WARNING: this feature in Alpha and the API is subject to change.
+
+Add a markdown button to your project's `README.md` to get your developers up and running with Coder with a few clicks.
+
+A basic example looks like this:
+
+```text
```

Comment:
I thought if I used `markdown` as the lang, the website would render as Markdown instead of showing the code but I don't think that's the case so I'll change this.